docs/Kubernetes/kubernetes.md

```diff
@@ -0,0 +1,70 @@
+### Introduction
+Note: Not for production use!!
+This guide with provide detail on how to deploy the AMTCloudToolkit services in Microsoft(C) Azure(R).
+Functionality has been added to MPS to allow it to scale and support a greater number of devices. 
+For this deployment, kubernetes will be used running in Azure(R) along with utilizing both redis and consul.
```

Comment:
"Scaling functionality in MPS enables Open AMT Cloud Toolkit to support a greater number of managed devices. For this deployment, kubernetes runs in Microsoft Azure* along with redis and consul."

What are redis and consul? Can you say a word about that?